README.md

```diff
@@ -10,10 +10,12 @@ This module consists of the following submodules:
 See more details in each module's README.
 
 ## Compatibility
-This module is meant for use with Terraform 0.13. If you haven't
-[upgraded](https://www.terraform.io/upgrade-guides/0-13.html) and need a Terraform
+This module is meant for use with Terraform 0.13+ and tested using Terraform 1.0+.
+If you find incompatibilities using Terraform `>=0.13`, please open an issue.
+
+If you haven't [upgraded][terraform-0.13-upgrade] and need a Terraform
```

Comment:
These reference links do no exist in the readme, we should add them.

Reply:
Done